docs: :memo: edits during review of README (#207)
dp-next/registers2parquet: README.qmd

## Installation

> To install from CRAN, run:

Reviewer: ```suggestion
Install from CRAN:
```

```{r, eval = FALSE}
install.packages("fastreg")
```

To install the latest development version from GitHub, run:

```{r, eval = FALSE}
# Development version on GitHub
pak::pak("dp-next/fastreg")
```

## Usage

> To convert a single SAS file to Parquet, use `convert_file()`:

Reviewer: This function also creates the Hive partition - add that here?

```{r, eval = FALSE}
library(fastreg)

convert_file(
  path = "path/to/file.sas7bdat",
  output_dir = "path/to/output_dir/"
)
```

You can use the helper function `list_sas_files()` to list all SAS files
in a directory, which can help convert several registers of the same

> year into one Parquet Hive partitioned dataset:

Reviewer: ```suggestion
Use `convert_register()` to convert several SAS files from the same register
into a Hive partitioned Parquet dataset. To list all SAS files in a directory, you can use the helper function `list_sas_files()`:
```

```{r, eval = FALSE}
convert_register(
  path = list_sas_files("path/to/sas_register/"),
  output_dir = "path/to/output_dir/"
)
```

fastreg also comes with a [targets](https://books.ropensci.org/targets/)

> template to make it easier to convert multiple registers in parallel:

Reviewer: ```suggestion
Use `use_targets_template()` to copy a [targets](https://books.ropensci.org/targets/) template that converts multiple registers in parallel into your project:
```

```{r, eval = FALSE}
use_targets_template()
```

> To read one of the Parquet registers, use `read_register()`:

Reviewer: ```suggestion
Use `read_register()` to read a Parquet register as a DuckDB table:
```
